src_base/xeve_enc.c

```diff
@@ -2618,7 +2618,7 @@ void xeve_param2string(XEVE_PARAM * param, char * sei_buf, int padx, int pady)
     sei_buf += snprintf(sei_buf, max_n, "profile=%d", param->profile);
     sei_buf += snprintf(sei_buf, max_n, " threads=%d", param->threads);
     sei_buf += snprintf(sei_buf, max_n, " input-res=%dx%d", param->w - padx, param->h - pady);
-    sei_buf += snprintf(sei_buf, max_n, " fps=%u", param->fps);
+    sei_buf += snprintf(sei_buf, max_n, " fps=%u", param->fps.num/ param->fps.den);
```

Comment:
this may need type casting param->fps.num to (float)param->fps.num

Reply:
Type casting is added.